packages/rules-unit-testing/src/api/index.ts

```diff
@@ -31,11 +31,16 @@ const DATABASE_ADDRESS_ENV: string = 'FIREBASE_DATABASE_EMULATOR_HOST';
 /** The default address for the local database emulator. */
 const DATABASE_ADDRESS_DEFAULT: string = 'localhost:9000';
 
-/** If any of environment variable is set, use it for the Firestore emulator. */
+/** If this environment variable is set, use it for the Firestore emulator. */
 const FIRESTORE_ADDRESS_ENV: string = 'FIRESTORE_EMULATOR_HOST';
 /** The default address for the local Firestore emulator. */
 const FIRESTORE_ADDRESS_DEFAULT: string = 'localhost:8080';
 
+/** If this environment variable is set, use it for the Storage emulator. */
+const STORAGE_ADDRESS_ENV: string = 'STORAGE_EMULATOR_HOST';
```

Comment:
This is the cloud variable which requires the protocol to be specified like `STORAGE_EMULATOR_HOST=http://localhost:9199` we could use the `FIREBASE_STORAGE_EMULATOR_HOST=localhost:9199` however this only works in new releases of the admin SDK